lib/autoupdate/agent/setup.go

```diff
@@ -307,14 +307,15 @@ func (ns *Namespace) Setup(ctx context.Context, path string, rev Revision, insta
 	return nil
 }
 
-func (ns *Namespace) installSELinux(ctx context.Context) error {
+func (ns *Namespace) installSELinux(ctx context.Context, rev Revision) error {
 	ns.log.InfoContext(ctx, "Installing SELinux module.")
 
 	if err := ns.checkSELinux(ctx, true); err != nil {
 		return trace.Wrap(err)
 	}
 
-	fileCtxs, err := selinux.FileContexts(ns.dataDir, ns.configFile)
+	binaryPath := filepath.Join(ns.Dir(), versionsDirName, rev.Dir(), "bin", "teleport")
```

Comment:
Does it matter that this path is not stable, and `teleport` is invoked via the `/usr/local/bin/teleport` symlink instead?